### ISSUE TITLE
Pre-Training the model?

### ISSUE THREAD

**Issue by Shikamaru5** [microsoft/GLIP#82]
Perhaps I'm missing something but what is the value in pre-training the model? From what I've read the teacher model is pre-trained in practically the exact same way as the student model. Wouldn't the model just given a huge dataset, overtime learn to gain those representations the same as with a teacher-student architecture? Since we deal in epochs it would likely just correct the wrong assumptions that it made originally and label better.
Also it's referenced in the paper a NLP noun parser, which was learnt about in the book 'Natural language processing with Python', is that from chapter 8.4 page 302, or is it sort of the whole book that gave the understanding for it?
